### PR TITLE
WIP: Dipole IO

### PR DESCRIPTION
**Pull request by agramfort**
more coming...

### CODE REVIEW

**Comment by Eric89GXL** [comment]
LGTM, I'll take another round to make Travis happy and then merge unless you're about to

**Comment by agramfort** [comment]
Let le finish the viz code first

**Comment by Eric89GXL** [comment]
I mean I can push fixes to your branch that make Travis happy.

Want 'em?

**Comment by agramfort** [comment]
go ahead but don't rebase

you have write access now

**Comment by Eric89GXL** [comment]
done

**Comment by agramfort** [comment]
thanks

**Comment by Eric89GXL** [comment]
@agramfort is it okay with you if I merge this for now? I want to go ahead with modifying `dipole.py` and not being able to rebase will be a pain.

File: mne/tests/test_dipole.py [hunk 1]
```diff
@@ -1,21 +1,50 @@
...
-    assert_true(len(time) == len(pos))
-    assert_true(len(time) == gof.size)
-    assert_true(len(time) == amplitude.size)
+    assert_true(len(times) == len(pos))
+    assert_true(len(times) == gof.size)
+    assert_true(len(times) == amplitude.size)
+
+    dipole = Dipole(times=times, pos=pos, amplitude=amplitude, ori=ori,
+                    gof=gof, name='ALL')
+    print(dipole)  # test repr
```
Comment (dengemann) [inline mne/tests/test_dipole.py]: a better test would be to use string formattig and check for some expected values in the string.

**Comment by dengemann** [comment]
LGTM!

**Comment by agramfort** [comment]
Ok. My time is up for today

File: mne/tests/test_dipole.py (hunk 1 above)
Comment (Eric89GXL) [inline mne/tests/test_dipole.py]: yes but @agramfort is lazy :)

**Comment by Eric89GXL** [comment]
Thanks

File: mne/tests/test_dipole.py (hunk 1 above)
Comment (agramfort) [inline mne/tests/test_dipole.py]: pragmatic ? or just too busy...

Comment (Eric89GXL) [inline mne/tests/test_dipole.py]: let's assume it's all three

Comment (agramfort) [inline mne/tests/test_dipole.py]: fine with me :)

**Pull request merged**